qa/L0_storage_GCS/test.sh

```diff
@@ -180,6 +205,47 @@ for MAYBE_SLASH in "" "/"; do
     done
 done 
 
+# Test with polling enabled
+SERVER_ARGS="--model-repository=$ROOT_REPO --exit-timeout-secs=120 --model-control-mode=poll"
+
+run_server
+if [ "$SERVER_PID" == "0" ]; then
+    echo -e "\n***\n*** Failed to start $SERVER\n***"
+    cat $SERVER_LOG
+    exit 1
+fi
+
+# copy contents of /models into S3 bucket and wait for them to be loaded.
+gsutil -m cp -r models/ "$BUCKET_URL_SLASH"
+sleep 120
+
+set +e
+
+# python unittest seems to swallow ImportError and still return 0
+# exit code. So need to explicitly check CLIENT_LOG to make sure
+# we see some running tests
+python $INFER_TEST >$CLIENT_LOG 2>&1
+if [ $? -ne 0 ]; then
+    cat $CLIENT_LOG
+    echo -e "\n***\n*** Test Failed\n***"
+    RET=1
+fi
+
+grep -c "HTTPSocketPoolResponse status=200" $CLIENT_LOG
```

Comment:
Is this still necessary after the change in unit test checking?

Reply:
I think the corresponding Python script need to inherit from `tu.TestResultCollector`

Reply:
Fixed. The script already does inherit from `tu.TestResultCollector`